net/net.go

```diff
@@ -718,7 +739,7 @@ func (n *net) subscribe(ctx context.Context, filter map[thread.ID]struct{}) (<-c
 }
 
 func (n *net) ConnectApp(a app.App, threadID thread.ID) (*app.Connector, error) {
-	info, err := n.getThread(context.Background(), threadID)
+	info, err := n.getThreadWithAddrs(threadID)
```

Comment:
Chose to use the address-augments `thread.Info` results here as well, not sure it matters.